pyrdfstore/store.py

```diff
@@ -310,8 +364,14 @@ def select(self, sparql: str, named_graph: Optional[str] = None) -> Result:
         )
         return target.query(sparql)
 
-    def insert(self, graph: Graph, named_graph: Optional[str] = None):
+    def insert(
+        self,
+        graph: Graph,
+        named_graph: Optional[str] = None,
+        safe_characters: Optional[str] = None,
```

Comment:
adding these on the signature is opening the door for a grwoing list of custom thingies like that

in #49 I took the root for:
- be able to set a cleaner at `__init__` of store, so you don't need to repeat at each call of `insert()`
- but make building cleaners somehting anyone could do (also a client, before sending graphs to store)